### PR TITLE
docs: update internalerror

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

remove the `{{page}}` macro

### Related issues and pull requests

part of #3893

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md [hunk 1]
```diff
@@ -3,54 +3,69 @@ title: InternalError
...
 ---
 
-{{JSRef}} {{non-standard_header}}
+{{JSRef}}{{Non-standard_Header}}
 
-**`InternalError` 对象**表示出现在 JavaScript 引擎内部的错误。例如： **"InternalError**: too much recursion"（内部错误：递归过深）。
+**`InternalError` 对象**表示出现在 JavaScript 引擎内部的错误。通常描述某种数量过多的情况，例如：
 
-## 语法
+- "too many switch cases"（过多`case`子句）；
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md:10]:
```suggestion
- "too many switch cases"（过多 `case` 子句）；
```

File: files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md [hunk 2]
```diff
@@ -3,54 +3,69 @@ title: InternalError
...
-## 语法
+- "too many switch cases"（过多 `case` 子句）；
+- "too many parentheses in regular expression"（正则表达式中括号过多）；
+- "array initializer too large"（数组初始化器过大）；
+- "too much recursion"（递归过深）。
 
-```plain
-new InternalError([message[, fileName[, lineNumber]]])
-```
+## 构造器
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md:15]:
```suggestion
## 构造函数
```

File: files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md [hunk 3]
```diff
@@ -3,54 +3,69 @@ title: InternalError
...
+- {{jsxref("Error.prototype.cause", "InternalError.prototype.cause")}}
+  - : 错误原因。从 {{jsxref("Error")}} 继承。
+- {{jsxref("Error.prototype.fileName", "InternalError.prototype.fileName")}} {{Non-standard_Inline}}
+  - : 引起错误的文件路径。从 {{jsxref("Error")}} 继承。
+- {{jsxref("Error.prototype.lineNumber", "InternalError.prototype.lineNumber")}} {{Non-standard_Inline}}
+  - : 引起错误位置的行编号。从 {{jsxref("Error")}} 继承。
+- {{jsxref("Error.prototype.columnNumber", "InternalError.prototype.columnNumber")}} {{Non-standard_Inline}}
+  - : 引起错误位置的列编号。从 {{jsxref("Error")}} 继承。
+- {{jsxref("Error.prototype.stack", "InternalError.prototype.stack")}} {{Non-standard_Inline}}
+  - : 堆栈跟踪信息。从 {{jsxref("Error")}} 继承。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md:35]: 可参考这个（直接复制替换即可）：https://github.com/mdn/translated-content/blob/6412b214f5dcf2e7a7821f6b71c1f58fe171e1ca/files/zh-cn/web/javascript/reference/global_objects/evalerror/index.md?plain=1#L19-L32

Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md:35]:
```suggestion
 - {{jsxref("Error.prototype.message")}} 
   - : 错误消息。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.name")}} 
   - : 错误名称。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.cause")}} 
   - : 表示导致当前错误被抛出的原因。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.fileName")}} {{non-standard_inline}} 
   - : 引发此错误的文件的路径。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.lineNumber")}} {{non-standard_inline}} 
   - : 引发此错误的代码所在的文件的行号。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.columnNumber")}} {{non-standard_inline}} 
   - : 引发此错误的代码在文件中所在行的列号。继承自 {{jsxref("Error")}}。 
 - {{jsxref("Error.prototype.stack")}} {{non-standard_inline}} 
   - : 堆栈跟踪。继承自 {{jsxref("Error")}}。
```

Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/internalerror/index.md:35]:
```suggestion
- {{jsxref("Error.prototype.message")}} 
  - : 错误消息。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.name")}} 
  - : 错误名称。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.cause")}} 
  - : 表示导致当前错误被抛出的原因。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.fileName")}} {{non-standard_inline}} 
  - : 引发此错误的文件的路径。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.lineNumber")}} {{non-standard_inline}} 
  - : 引发此错误的代码所在的文件的行号。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.columnNumber")}} {{non-standard_inline}} 
  - : 引发此错误的代码在文件中所在行的列号。继承自 {{jsxref("Error")}}。 
- {{jsxref("Error.prototype.stack")}} {{non-standard_inline}} 
  - : 堆栈跟踪。继承自 {{jsxref("Error")}}。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**